### ISSUE TITLE
Shortcut for temp keymap

### ISSUE THREAD

**Issue by GL513** [keymapperorg/KeyMapper#674]
When the shortcut is launched, it overrides the current keymap, if any. If another shortcut was launched with the same trigger (keymap), it would either have to wait for the first one to finish, or it would not run, depending on the user's desire.

**Comment by GL513** [comment]
Yeah i accedently closed this lol

**Comment by sds100** [comment]
What do you mean by shortcut? An app shortcut?

**Comment by GL513** [comment]
> What do you mean by shortcut? An app shortcut?

Yes, a shortcut. I was just thinking as soon as i opened this, mabey add a custom timeout as well.

**Comment by sds100** [comment]
I'm still confused by what this feature would do. Can you share a screen recording of the current (wrong) behaviour.

**Comment by GL513** [comment]
> I'm still confused by what this feature would do. Can you share a screen recording of the current (wrong) behaviour.
This is a feature i would like to see in the future. No problems with it at the moment.
